prototype/core/impl/src/test/java/org/eclipse/sensinact/prototype/command/impl/GatewayThreadImplTest.java

```diff
@@ -69,28 +65,31 @@ void teardown() {
         thread.deactivate();
     }
 
-    @Test
-    void testExecute() throws Exception {
+    private static final int DELAY = 100;
+    private static final int TEST_VALUE = 5;
+
+    @ParameterizedTest
+    @ValueSource(ints = { 1, 10, 100, 1000 })
```

Comment:
I don't think that 1 and 10 are good values to use here. If the CI server is heavily loaded then we are likely to see timing related failures and "flaky tests" for the line:
 
    assertFalse(result.isDone())

I'd prefer not to remove this test expression as it guarantees that the command is run asynchronously, hence removing the very short time delays.

Reply:
ok